gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -1390,6 +1415,32 @@ gst_tensor_transform_stand (GstTensorTransform * filter,
   return GST_FLOW_OK;
 }
 
+/**
+ * @brief subrouting for tensor-tranform, "clamp" case.
+ *        : pixel = if (pixel > max) ? max :
+ *                  if (pixel < min) ? min : pixel
+ * @param[in/out] filter "this" pointer
+ * @param[in] inptr input tensor
+ * @param[out] outptr output tensor
+ * @return Gst flow status
+ */
+static GstFlowReturn
+gst_tensor_transform_clamp (GstTensorTransform * filter,
+    const uint8_t * inptr, uint8_t * outptr)
+{
+  uint32_t *fromDim = filter->in_config.info.dimension;
+  size_t Size = fromDim[3] * fromDim[2] * fromDim[1] * fromDim[0];
+  float *in = (float *) inptr;
```

Comment:
Q: Why is float used here instead of NNS_TYPE *in = (NNS_TYPE *) inptr?

Reply:
I just took the similar codes to `gst_tensor_transform_stand`, but yes, they should be changed.